src/components/Link/src/Link.stories.mdx

```diff
@@ -0,0 +1,194 @@
+import { Link } from "@gemeente-denhaag/link";
+import { Paragraph } from "@gemeente-denhaag/typography";
+const ReactParagraph = (
+  <Paragraph>
+    It is possible to put <Link href="#" disabled>a link</Link> in a Paragraph element.
+  </Paragraph>
+);
+    `,
+        },
+      },
+    }}
+  >
+    {(args) => (
+      <Paragraph>
+        It is possible to put{" "}
+        <Link href="#" {...args}>
+          a link
+        </Link>{" "}
+        in a Paragraph element.
+      </Paragraph>
+    )}
+  </Story>
+</Canvas>
+
+## Icon Start
+
+<Canvas>
+  <Story
+    name="Icon Start"
+    args={{ icon: <ArrowLeftIcon />, iconAlign: "start" }}
+    parameters={{
+      docs: {
+        source: {
+          code: `
+import { Link } from "@gemeente-denhaag/link";
+import { Paragraph } from "@gemeente-denhaag/typography";
+import { ArrowLeftIcon } from "@gemeente-denhaag/icons";
+const ReactParagraph = (
+  <Paragraph>
+    It is possible to put <Link href="#" icon={<ArrowLeftIcon />} iconAlign="start">a link</Link> in a Paragraph element.
+  </Paragraph>
+);
+    `,
+        },
+      },
+    }}
+  >
+    {(args) => (
+      <Paragraph>
+        It is possible to put{" "}
+        <Link href="#" {...args}>
+          a link
+        </Link>{" "}
+        in a Paragraph element.
+      </Paragraph>
+    )}
+  </Story>
+</Canvas>
+
+### Icon End
+
+<Canvas>
+  <Story
+    name="Icon End"
+    args={{ icon: <ArrowRightIcon /> }}
+    parameters={{
+      docs: {
+        source: {
+          code: `
+import { Link } from "@gemeente-denhaag/link";
+import { Paragraph } from "@gemeente-denhaag/typography";
+import { ArrowRightIcon } from "@gemeente-denhaag/icons";
+const ReactParagraph = (
+  <Paragraph>
+    It is possible to put <Link href="#" icon={<ArrowRightIcon />}>a link</Link> in a Paragraph element.
+  </Paragraph>
+);
+    `,
+        },
+      },
+    }}
+  >
+    {(args) => (
+      <Paragraph>
+        It is possible to put{" "}
+        <Link href="#" {...args}>
+          a link
+        </Link>{" "}
+        in a Paragraph element.
+      </Paragraph>
+    )}
+  </Story>
+</Canvas>
+
+### HTML only Link
+
+<Canvas>
+  <Story name="HTML only Link">
+    <p class="utrecht-paragraph utrecht-paragraph--distanced">
+      It is possible to put{" "}
+      <a href="#" class="denhaag-link denhaag-link--with-icon" tabindex="0">
+        <span class="denhaag-link__icon denhaag-link__icon--start">
+          <svg
+            width="1em"
+            height="1em"
+            viewBox="0 0 24 24"
+            fill="none"
+            xmlns="http://www.w3.org/2000/svg"
+            class="MuiSvgIcon-root"
+            focusable="false"
+            aria-hidden="true"
+            shape-rendering="auto"
+          >
+            <path
+              d="M11.707 18.707a1 1 0 01-1.414 0l-6-6a1 1 0 010-1.414l6-6a1 1 0 111.414 1.414L7.414 11H19a1 1 0 110 2H7.414l4.293 4.293a1 1 0 010 1.414z"
+              fill="currentColor"
+            ></path>
```

Comment:
Would it be possible to include all variants in the html only story. It'd be easier for people only copying our bem structure.